docs/ExpectAPI.md

```diff
@@ -416,15 +416,15 @@ For example, this code tests that the promise rejects with reason `'octopus'`:
 ```js
 test('rejects to octopus', () => {
   // make sure to add a return statement
-  return expect(Promise.reject('octopus')).rejects.toBe('octopus');
+  return expect(Promise.reject(new Error('octopus'))).rejects.toThrow('octopus');
```

Comment:
This old example is still valid, right?

Reply:
(although it's not recommended to reject with a non-error)

Reply:
yes, it still works